src/dscim/preprocessing/preprocessing.py

```diff
@@ -301,107 +301,3 @@ def subset_USA_ssp_econ(
         consolidated=True,
         mode="w",
     )
-
-
-def clip_damages(
```

Comment:
I didn't find reference to this function anywhere except `dscim-cil` which only references it in [this](https://github.com/ClimateImpactLab/dscim-cil/blob/f613b0aeb12a7fe9addda6de9a700733451c81ab/misc_scripts/master_script.sh) script. If this script were updated to be operational, this function would be removed entirely.

Reply:
@JMGilbert can you create a new issue about removing this, then do a new PR that removes this function entirely? Then we can reference the PR in dscim-cil where your link points. Thank you!

Reply:
Noting here that we decided this doesn't need an Issue. It's noted in the CHANGELOG